main/src/vtr_pose_graph/src/serializable/rc_graph.cpp

```diff
@@ -138,11 +138,8 @@ void RCGraph::saveGraphIndex() {
 
 void RCGraph::saveVertices() {
   // save any unsaved data first
-  CLOG(DEBUG, "pose_graph") << "Saving any unsaved data in cache";
-  for (auto iter = vertices_.begin(); iter != vertices_.end(); ++iter)
-    iter->second->unload();
-
-  CLOG(DEBUG, "pose_graph") << "Saving vertices to disk";
+  for (auto iter = vertices_.begin(); iter != vertices_.end(); ++iter){
+    iter->second->unload();}
```

Comment:
Could this closing curly brace be moved to the next line (or deleted)?

Reply:
Removed it and added the debugging print statement back in - not sure why it was removed.